### DISCUSSION TITLE
Configuring upgrades for particular aliased npm packages

### DISCUSSION THREAD

**Discussion by glasser** [renovatebot/renovate#17145] [Request Help]
I am starting to use the npm package alias feature in my project, which Renovate supports (#3577). So I have this in the `dependencies` block of a package.json:

```
    "apollo-gateway-0-oldest-supported": "npm:@apollo/gateway@0.50.1",
    "apollo-gateway-0-before-asgi": "npm:@apollo/gateway@0.51.0",
    "apollo-gateway-0-latest": "npm:@apollo/gateway@0.52.0",
    "apollo-gateway-2-oldest-supported": "npm:@apollo/gateway@2.0.0",
    "apollo-gateway-2-before-asgi": "npm:@apollo/gateway@2.0.5",
    "apollo-gateway-2-latest": "npm:@apollo/gateway@2.1.0",
```

What I'd like Renovate to do is:
- keep `apollo-gateway-0-latest` updated to the latest 0.x release
- keep `apollo-gateway-2-latest` updated to the latest release overall
- keep the other four pinned at exactly those releases

How can I specify this? Can I use `matchPackageNames` with the string like `apollo-gateway-0-latest`?

**Comment by viceice** [comment]
sure, i think it should work
